secretsauce/apps/account/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+# TODO: hash token field
+class User(AbstractUser):
+    email = models.EmailField(verbose_name='email', max_length=255, unique=True)
+    phone = models.CharField(null=True, max_length=255)
+    token = models.CharField(max_length=60, null=False)
+    token_class = models.CharField(max_length=255)
```

Comment:
yes, token means the RMS token. I will change the field to OneToOneField

Reply:
What is token_class

Reply:
it means the client group (like McDonald Australia) the user belongs to

Reply:
token_class is both repeated in User and Token model. I would suggest removing the field on user model. Only keep it on Token model.

Additionally, based on our UML diagram, you can see there's quite a fair bit of overlap in terms of the idea of a "client group"
1. token_class in user models here
2. Company in client_user model in the UML diagram
3. Organization in project model in the UML diagram

We need to do some editing of the class diagrams afterwards

![image](https://user-images.githubusercontent.com/39354695/83090863-5bac1c00-a0cc-11ea-90bd-94b66094259e.png)

Reply:
On hindsight, the token model should not even need to know which client group the client is in i.e. what additional benefit does it have of doing so?
Need @gorold to follow up on this in the future

Reply:
my opinions: 
the token class has the company field is because we want RMS user (admin) to decide the user can access which projects (I have renamed the token_class to Company)

the "organization" field in Project class should also refer to Company